.circleci/common.sh

```diff
@@ -62,10 +62,7 @@ function install_deps_pytorch_xla() {
   # Bazel doesn't work with sccache gcc. https://github.com/bazelbuild/bazel/issues/3642
   sudo apt-get -qq update
 
-  # Install npm, nodejs, and necessary dependencies
-  sudo apt-get -qq install -y aptitude
```

Comment:
We don't need this anymore?

Reply:
So all the CircleCI related changes were part of the `revert` commit. Whatever that's deleted/removed, we're just reverting this one commit -- https://github.com/pytorch/xla/commit/91b5b55575fd249edd7b3cb852c659fc8655a35f.